aidants_connect_web/templates/aidants_connect_web/users_search_bar.html

```diff
@@ -0,0 +1,14 @@
+{% comment %}
+This search bar is designed to be used with JS logic in
+aidants_connect_web/static/js/users_search.js
+
+Just add it anywhere you display users along with the following static files:
+- aidants_connect_web/static/css/users_search.css
+- aidants_connect_web/static/js/users_search.js
+{% endcomment %}
+<div class="user-search row" hidden>
+  <label id="filter-input-label" for="filter-input">
+    Rechercher un usager ou une usagère
+  </label>
+  <input id="filter-input" type="text" aria-describedby="filter-input-label" />
```

Comment:
Pas besoin du `aria-describedby` ici, le `for` sur le `label` suffit :
```suggestion
  <input id="filter-input" type="text" />
```

`aria-describedby` a plutôt vocation à contenir les id des éléments qui contiendraient des messages d'erreur ou d'une information supplémentaire pour remplir le champ (les contraintes d'un mot de passe ou le format attendu de la date de naissance par exemple). Donc typiquement tu mettrais `aria-describedby="filter-input-error filter-input-help"` par exemple.

En l'état je dirais que le lecteur d'écran dirait 2 fois "rechercher un usager ou une usagère" : une fois parce que c'est le label, une fois parce que c'est la description.